GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTNanopbHelpers.m

```diff
@@ -136,8 +138,13 @@ gdt_cct_LogEvent GDTCCTConstructLogEvent(GDTCOREvent *event) {
   logEvent.has_event_uptime_ms = 1;
   logEvent.timezone_offset_seconds = event.clockSnapshot.timezoneOffsetSeconds;
   logEvent.has_timezone_offset_seconds = 1;
-  // TODO: Read network_connection_info from the custom params dict.
-
+  if (event.customPrioritizationParams[@"network_connection_info"]) {
+    NSData *networkConnectionInfoData =
+        event.customPrioritizationParams[@"network_connection_info"];
+    [networkConnectionInfoData getBytes:&logEvent.network_connection_info
+                                 length:sizeof(logEvent.network_connection_info)];
```

Comment:
I think length should be networkConnectionInfoData.length. The reason why this works is because the network_connection_info struct has no dynamic member variables and has a static length of 13 bytes.